pyqtgraph/graphicsItems/DateAxisItem.py

```diff
@@ -0,0 +1,319 @@
+import sys
+import numpy as np
+import time
+from datetime import datetime, timedelta
+
+from .AxisItem import AxisItem
+from ..pgcollections import OrderedDict
+
+__all__ = ['DateAxisItem', 'ZoomLevel']
```

Comment:
I'd rather remove 'ZoomLevel' ;)

Reply:
Speaking of which: We advertise overriding `DateAxisItem.zoomLevels`, although the entries of that dict, such as `YEAR_MONTH_ZOOM_LEVEL`, are not available through `__all__`.

Reply:
@2xB Does overriding `zoomLevels` have any practical purpose except for the locale representation of dates (for example someone might like "04.12" for the 4th of December instead of "4 Mon")? Also the names of the levels has become somewhat misleading: "HOUR_MINUTE_ZOOM_LEVEL" level stands for "day minute" actually and "HMS_ZOOM_LEVEL" stands for "second" in fact.

Reply:
The concept of a user-overridable `zoomLevels` dict actually comes from @3rdcycle in the PR #74 . I think the idea was that at that point, the zoom levels were not as precise as they are now, so there would be a good cause for the user to play around with the `zoomLevels` dict to make spacing better. Since we are now even DPI-aware, I would agree to you that there is probably no practical use to that and suggest to just remove this from the docstrings.